careerfairsystems/arkad-app-2018
# Add company and event details screen

## Change description

- Add company details screen
  - Adding all relevant info from API
- Fix sorting on company list
- Add event details screen

## How to verify

Run the project and:
- Go to the Companies tab
  - Check if it's sorted alphabetically
- Visit different companies and
  - Check that a logo exist for each company
  - Check that relevant information is shown
- Go to the Events tab and visit different events
  - Confirm that you are taken to a new view when pressing a event

## Issues fixed

This fixes #30 , fixes #33, fixes #27.
